sql/core/src/test/scala/org/apache/spark/sql/TPCDSSchema.scala

```diff
@@ -0,0 +1,259 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+trait TPCDSSchema {
+
+  private val tableColumns = Map(
+    "call_center" ->
```

Comment:
@dbaliafroozeh . I'm just wondering  you changed the order of tables from the original code? For me,
1. This is not consistent with the original code (It starts with `catalog_page` table.)
    - This is the best for review.
2. This is not consistent with TPCDS spec document. (Usually, it starts with `Fact Table Definition`: `store_sales` table)
    - This is the best to compare with TPCDS spec doc.
3. This is not alphabetic.
    - This is the best for human readability and to lookup on GitHub.

While you are here, could you choose one of the above and update this PR?

Reply:
I have updated the list of tables according to the TPCDS spec (option 2). In addition of being consistent with the spec, having fact tables `_sales` and `_return` next to each other looked most logical to me.

Reply:
Could you include the link to the PR description? It's always good to explain the way of your thought.